modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/ImportStyleBookDisplayContext.java

```diff
@@ -23,48 +23,46 @@ public ImportStyleBookDisplayContext(RenderRequest renderRequest) {
 		_renderRequest = renderRequest;
 	}
 
-	public List<String> getStyleBookEntryZipProcessorImportResultEntryNames(
-		StyleBookEntryZipProcessorImportResultEntry.Status status) {
+	public List<String> getStyleBookImporterResultEntryNames(
+		StyleBookEntryImporterImportResultEntry.Status status) {
 
-		List<StyleBookEntryZipProcessorImportResultEntry>
-			styleBookEntryZipProcessorImportResultEntries =
-				_getStyleBookEntryZipProcessorImportResultEntryNames();
+		List<StyleBookEntryImporterImportResultEntry>
+			styleBookImporterResultEntries =
+				_getStyleBookImporterResultEntryNames();
 
-		if (ListUtil.isEmpty(styleBookEntryZipProcessorImportResultEntries)) {
+		if (ListUtil.isEmpty(styleBookImporterResultEntries)) {
 			return null;
 		}
 
 		return TransformUtil.transform(
-			styleBookEntryZipProcessorImportResultEntries,
-			styleBookEntryZipProcessorImportResultEntry -> {
-				if (styleBookEntryZipProcessorImportResultEntry.getStatus() !=
+			styleBookImporterResultEntries,
+			styleBookEntryImporterImportResultEntry -> {
+				if (styleBookEntryImporterImportResultEntry.getStatus() !=
 						status) {
 
 					return null;
 				}
 
-				return styleBookEntryZipProcessorImportResultEntry.getName();
+				return styleBookEntryImporterImportResultEntry.getName();
 			});
 	}
 
-	private List<StyleBookEntryZipProcessorImportResultEntry>
-		_getStyleBookEntryZipProcessorImportResultEntryNames() {
+	private List<StyleBookEntryImporterImportResultEntry>
+		_getStyleBookImporterResultEntryNames() {
```

Comment:
Since you're refactoring this method should be renamed to match it's return type: `_getStyleBookEntryImporterImportResultEntries`

It was incorrectly renamed [here](https://github.com/liferay/liferay-portal/commit/35dfd327c042ad660727772e56804a64a2801e73).